Doc/library/logging.rst

```diff
@@ -1151,37 +1151,40 @@ functions.
    |              | using the specified filename, rather than a |
    |              | StreamHandler.                              |
    +--------------+---------------------------------------------+
-   | ``filemode`` | Specifies the mode to open the file, if     |
-   |              | filename is specified (if filemode is       |
-   |              | unspecified, it defaults to 'a').           |
+   | ``filemode`` | If ``filename`` is specified, open the file |
+   |              | in this :ref:`mode <filemodes>`. Defaults   |
+   |              | to ``'a'``.                                 |
    +--------------+---------------------------------------------+
    | ``format``   | Use the specified format string for the     |
    |              | handler.                                    |
    +--------------+---------------------------------------------+
-   | ``datefmt``  | Use the specified date/time format.         |
+   | ``datefmt``  | Use the specified :func:`time.strftime`     |
+   |              | style date/time format.                     |
```

Comment:
I suggest a different wording, to avoid the non-hyphenated use of "style", e.g.:

"Use the specified date/time format, as accepted by `time.strftime`."